lib/windows/winlean.nim

```diff
@@ -36,6 +36,8 @@ type
   DWORD* = int32
   PDWORD* = ptr DWORD
   LPINT* = ptr int32
+  ULONG_PTR* = uint
```

Comment:
> Remember last bug, where we can't use ports larger than 32768 because port was declared as int16 not uint16. 

That was not an oversight, back then I read somewhere that larger ports are not supported anyway and back then Nim didn't have uint16.